docs/api/1.x/filtering.rst

```diff
@@ -76,48 +83,59 @@ types is not defined. For example, if you have a record like
 not. However, a comparison operator will match whatever order you get
 by sorting, and the ordering will include all records.
 
-**Multiple values**
+Multiple values
+---------------
 
 Prefix field with ``in_`` and provide comma-separated values.
 
 * ``/collection?in_status=1,2,3``
 
-**Exclude**
+Exclude
+-------
 
 Prefix field name with ``not_``:
 
 * ``/collection?not_field=0``
 
-**Exclude multiple values**
+Exclude multiple values
+-----------------------
 
 Prefix field name with ``exclude_``:
 
 * ``/collection?exclude_field=0,1``
 
-**Search string fields**
+Search string fields
+--------------------
 
 Prefix field name with ``like_``:
 
 * ``/collection?like_field=foo``
 
-**Field existence**
+The specified value can also contain wildchars:
+
+* ``/collection?like_field=foo*`` (starts with ``foo``)
+* ``/collection?like_field=*foo`` (ends with ``foo``)
+* ``/collection?like_field=*foo*`` (equivalent to ``like_field=foo``)
```

Comment:
Except if I've miread the patch, I'd probably add a warning about the fact that now searching for a field string actually containing the `*` is now impossible.